.github/pull_request_template.md

```diff
@@ -1,18 +1,22 @@
-*Before you submit this PR make sure you have added an entry to **.changelog** file.*
+# Some hints
+
+Consider adding an entry to the `.changelog` file in the toplevel directory.
+It is a hidden file with a dot at the beginning.
```

Comment:
Since it seems there is no more `.changelog` but a `CHANGELOG.md` this paragraph probably needs to be adjusted.

Reply:
Yes, my bad. I plan to create automated notes for our releases. I am using a GitHub action that reads the CHANGELOG.md and extracts them. WIP.

Reply:
@jcayouette For clarification, in `master` we currently already use `CHANGELOG.md` and in the 4.3 it is still `.changelog`?

Reply:
I changed it accordingly.  @jcayouette can you please check again, and approve if appropriate?